test/floating_test.dart

```diff
@@ -0,0 +1,11 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_speed_dial/flutter_speed_dial.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:generator/home/homePage.dart';
+
+void main() {
+  testWidgets('finds a Widget', (WidgetTester tester) async {
```

Comment:
**issue**: This test is failing for me
```
A Timer is still pending even after the widget tree was disposed.
Failed assertion: line 1278 pos 12: '!timersPending'
```

Reply:
**suggestion:** naming tests to be something more descriptive by utilizing both group and test names helps significantly as a project grows